### PR TITLE
doc: Highlight DNS requests part in tor.md

### PR DESCRIPTION
**Pull request by ghost**
_What?_

Highlight DNS requests part in Proxy section

_Why?_

1. DNS requests are very important while considering privacy
2. Lot of users might skip reading it because of the way it is mixed with everything else in the doc right now
3. I have seen lot of users ignoring DNS requests or unaware of such things while using privacy tools

_How?_

Initially I had tried keeping these lines separate from code block but [Jonatack didn't agree with the changes](https://github.com/bitcoin/bitcoin/pull/21157#discussion_r618177116). Harding suggested using [bold/italic in `<pre></pre>`](https://github.com/bitcoin/bitcoin/pull/21157#discussion_r644364143). I have used the suggestions from previous PR and added `---`

This is a part of alternative described in https://github.com/bitcoin/bitcoin/pull/22316

### CODE REVIEW

**Comment by Rspigler** [comment]
ACK 9bcea02c5b5fb3d8c622a4a0dce20a4ab8961d99

**Review by kristapsk — APPROVED** [review]
ACK 9bcea02c5b5fb3d8c622a4a0dce20a4ab8961d99

File: doc/tor.md [hunk 1]
```diff
@@ -41,9 +41,11 @@ outgoing connections, but more is possible.
     -onion=ip:port  Set the proxy server to use for Tor onion services. You do not
                     need to set this if it's the same as -proxy. You can use -onion=0
                     to explicitly disable access to onion services.
-                    Note: Only the -proxy option sets the proxy for DNS requests;
+                    ------------------------------------------------------------------
+                    <b>Note: Only the -proxy option sets the proxy for DNS requests;
                     with -onion they will not route over Tor, so use -proxy if you
-                    have privacy concerns.
+                    have privacy concerns.</b>
+                    ------------------------------------------------------------------
```
Comment (jonatack) [inline doc/tor.md:48]: Maybe just bold or just the horizontal separators.  Adding emphasis is noisy and eventually, as more is added for well-intentioned handholding, it makes the doc harder to read and understand and has to be tuned out by readers.  A clean doc is a nice thing.

Comment (prayank23) [reply doc/tor.md:48]: Okay. Will remove bold.

Comment (jonatack) [reply doc/tor.md:48]: Thanks!

**Comment by jonatack** [comment]
Thanks for updating.

ACK 86a4a15bdcc96eb565ab80166642d71d542061a9

**Review by RiccardoMasutti — APPROVED** [review]
ACK 86a4a15

**Comment by Rspigler** [comment]
ACK 86a4a15bdcc96eb565ab80166642d71d542061a9

**Review by lsilva01 — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/22317/commits/86a4a15bdcc96eb565ab80166642d71d542061a9

**Review by kristapsk — APPROVED** [review]
ACK 86a4a15bdcc96eb565ab80166642d71d542061a9

**Review by theStack — APPROVED** [review]
ACK 86a4a15bdcc96eb565ab80166642d71d542061a9

Agree that this privacy-relevant part deserves to be highlighted 🕵️

**Comment by prayank23** [comment]
@MarcoFalke @fanquake @laanwj I can get more 10 ACKs here more this two lines change in doc. Sorry but I am not interested anymore.

**Comment by Rspigler** [comment]
We're all on the same team.  If you decide not to reopen as discussed on IRC, I can pickup.

**Comment by prayank23** [comment]
Reopened after reading this question: https://bitcoin.stackexchange.com/questions/111801/restrict-bitcoin-core-to-tor/

Change still makes sense, respect reviewers for their time to review and agree, hopeful this could get merged and improve docs.

**Comment by katesalazar** [comment]
I think this isn't a bad (nor a good) idea.

Because the Don't Repeat Yourself principle,
the affected lines shouldn't be edited manually,
but be assembled from their true sources
residing somewhere else.

**Comment by prayank23** [comment]
image

I never realized this is mentioned somewhere in docs until I read it in one of the comments in some pull request. DNS is obviously important if discussing privacy so which doc would work better for users: 1 or 2?

**Comment by katesalazar** [comment]
> which doc would work better for users: 1 or 2?

for many users who would value better that doc be
perfectly synced to the program output, 2.

**Comment by prayank23** [comment]
> > which doc would work better for users: 1 or 2?
> 
> for many users who would value better that doc be perfectly synced to the program output, 2.

Thanks for sharing your opinion. I disagree though and this doc can be improved a lot. This is one of the improvements which had some ACKs.

**Comment by achow101** [comment]
ACK 86a4a15bdcc96eb565ab80166642d71d542061a9

**Pull request merged**